package.json

```diff
@@ -56,7 +56,7 @@
     "react-transform-hmr": "^1.0.4",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^1.14.0",
+    "webpack": "2",
```

Comment:
Ah cool, I guess it’d only write it if the requested version no longer matched the one in `Yarn.lock`. Awesome! 😃